content/blog/sprint-review-2102.md

```diff
@@ -0,0 +1,54 @@
+Title: Wazo Platform 21.02 Released
+Date: 2021-02-15
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2102
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 21.02 release.
+
+## New features in this release
+
+* **Call Recording**: Call recordings are now attached to the call's CDR. There's an API to fetch the media from a given CDR.
```

Comment:
Could be nice to add a link to the new API reference or atleast to CDR section : https://wazo-platform.org/documentation/api/cdr.html#tag/cdr